clang/lib/DPCT/ExternalReplacement.cpp

```diff
@@ -161,6 +171,9 @@ int mergeExternalReps(std::string InRootSrcFilePath,
   std::vector<std::pair<std::string, std::string>> MainSrcFilesDigest;
   std::map<std::string, std::vector<clang::tooling::CompilationInfo>>
       CompileTargets;
+  auto Hash = llvm::sys::fs::md5_contents(InRootSrcFilePath);
+  MainSrcFilesDigest.push_back(
+          std::make_pair(InRootSrcFilePath, Hash->digest().c_str()));
```

Comment:
".c_str()" is not needed.

Reply:
because `Hash->digest()` is `llvm::SmallString`, it need to convert to `char*` to put into string vector.